package.json

```diff
@@ -45,6 +45,10 @@
 				"pester.runTestsInNewProcess": {
 					"type": "boolean",
 					"markdownDescription": "This will cause all Pester Test invocations to occur each time in a new process, rather than reuse a runspace. This is useful if you are testing Powershell Classes or Assemblies that cannot be unloaded or reloaded, but comes at a significant performance cost. This currently has no effect on debugging tests."
+				},
+				"pester.pesterModulePath": {
```

Comment:
@JustinGrote I built the vsix and installed it in VS Code Insiders. But this setting did not popup in the GUI. Must I do something more?

Reply:
Ignore that. I click "install recommended extensions" which overwrote with the one from the marketplace. :)